core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -109,7 +109,7 @@ class CudaInternal {
   mutable std::size_t m_scratchUnifiedCount;
   mutable std::size_t m_scratchFunctorSize;
 
-  size_type m_scratchUnifiedSupported;
+  bool m_scratchUnifiedSupported;
```

Comment:
I am unsure about this change due to alignment stuff. But I guess we don't generally avoid bool members.